code/datums/uplink_item.dm

```diff
@@ -835,6 +835,14 @@ var/list/uplink_items = list()
 	discounted_cost = 2
 	jobs_with_discount = list("Chef")
 
+/datum/uplink_item/jobspecific/service/specialsauce
```

Comment:
Forgot to change this. Should be
/datum/uplink_item/jobspecific/service/wendigomeat